strings/task_02.py

```diff
@@ -0,0 +1,47 @@
+excerpt = '''У лукоморья дуб зелёный;
+Златая цепь на дубе том:
+И днём и ночью кот учёный
+Всё ходит по цепи кругом;
+Идёт направо — песнь заводит,
+Налево — сказку говорит.
+Там чудеса: там леший бродит,
+Русалка на ветвях сидит;
+Там на неведомых дорожках
+Следы невиданных зверей;
+Избушка там на курьих ножках
+Стоит без окон, без дверей;
+Там лес и дол видений полны;
+Там о заре прихлынут волны
+На брег песчаный и пустой,
+И тридцать витязей прекрасных
+Чредой из вод выходят ясных,
+И с ними дядька их морской;
+Там королевич мимоходом
+Пленяет грозного царя;
+Там в облаках перед народом
+Через леса, через моря
+Колдун несёт богатыря;
+В темнице там царевна тужит,
+А бурый волк ей верно служит;
+Там ступа с Бабою Ягой
+Идёт, бредёт сама собой,
+Там царь Кащей над златом чахнет;
+Там русский дух… там Русью пахнет!
+И там я был, и мёд я пил;
+У моря видел дуб зелёный;
+Под ним сидел, и кот учёный
+Свои мне сказки говорил.'''
+
+splited_excerpt = excerpt.split()
+join_excerpt = ''.join(splited_excerpt)
+letters_list = []
+words_list = []
+
+for letter in join_excerpt[2::3]:
+    if letter.isalpha():
+        letters_list.append(letter)
+for words in splited_excerpt[1::2]:
+    if words != '—':
+        words_list.append(words)
+print('every third letter ->', letters_list)
```

Comment:
let's not create new lists when the customer does not ask for it =) and at the end you have lists in the output, I asked for other data types